.github/workflows/tests.yml

```diff
@@ -43,5 +43,11 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install tox tox-gh-actions
+      - name: Lint with tox
+        run: tox -e lint
+      # - name: Format with tox
+      #   run: tox -e format
+      - name: Flake8 with tox
+        run: tox -e flake8
       - name: Test with tox
-        run: tox
+        run: tox -e test
```

Comment:
Since we're at it maybe we can also add the workflow to deploy the package to pypi using the t`tox` build commands?
`tox -e build && tox -e release`

Maybe we can use this [github](https://github.com/pypa/gh-action-pypi-publish) action with a manual approval step using a "release" environment? https://cloudlumberjack.com/posts/github-actions-approvals/

Would be totally OK if we leave this for another PR

Reply:
I think there is build & release script already added to the `tox.ini` file by @rbavery.

Reply:
I think Rodrigo's method sounds like a good option to switch to, I don't think I added the build and release script(it was already there). It'd be great to handle this with a gh action with a manual approval step

Reply:
@srmsoumya do you need pypi credentials to authenticate this manual progress? @avanetten do you have these available and could you share them in an email?

Reply:
Added a release workflow here #449

Reply:
No Manual step for now but it will only run if you add a tag to the PR that increases the version.